src/query.rs

```diff
@@ -265,26 +265,29 @@ where
                 })
                 .collect();
 
+            let has_next_page = if pages > 0 { page < pages - 1 } else { false };
             let start_cursor = edges.first().map(|edge| edge.cursor.clone());
             let end_cursor = edges.last().map(|edge| edge.cursor.clone());
 
             Ok(Connection {
                 edges,
                 page_info: PageInfo {
                     has_previous_page: page != 1,
-                    has_next_page: page < pages - 1,
+                    has_next_page,
                     start_cursor,
                     end_cursor,
                 },
                 pagination_info: Some(PaginationInfo {
                     pages,
                     current: page,
                     offset: page * limit,
-                    total: pages * limit
+                    total: pages * limit,
                 }),
             })
         } else if let Some(offset_object) = offset_object {
-            let offset_object = offset_object.object().expect("We expect Offset to be object");
+            let offset_object = offset_object
+                .object()
+                .expect("We expect Offset to be object");
```

Comment:
Did cargo fmt somewhen just changed, or we are using different version?

Reply:
yes I will remove all formatting changes, only just noticed them. my bad.